### ISSUE TITLE
Multiple intervals within the same weekday in blocked services schedule

### ISSUE THREAD

**Issue by venethia** [AdguardTeam/AdGuardHome#6326]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Id' like to suspend a blocked service more times the same day.

Example: from 8:00 to 14:00 and from 20:00 to 23:59.

### Proposed solution

Add one more interval both on global blocked services and on per client blocked services after adding a new client.

**Comment by Burrito78** [comment]
I just want to unblock YouTube on the AppleTV from the evening 21:00pm - 03:00am in the night. This isn't possible currently because only one schedule is allowed per day and "end time needs to be before start time".

**Comment by y8s** [comment]
Yes please. Can't block a time spanning midnight, only from numerically lower times to numerically higher times and a single schedule per persistent client.

**Comment by neill12** [comment]
+1 for this

**Comment by stuffu** [comment]
Yes please!

**Comment by lukethacoder** [comment]
Would be a handy feature to "bulkify" the intervals config. Looks like there might be a bit of work involved as the schema currently only supports a `start`/`end` time, and not an array of `start`/`end` time values.

https://github.com/AdguardTeam/AdGuardHome/blob/fbc4a2700773aa30e780f1b518f47c639ea7a4c4/openapi/openapi.yaml#L2654-L2701

https://github.com/AdguardTeam/AdGuardHome/blob/fbc4a2700773aa30e780f1b518f47c639ea7a4c4/client/src/initialState.ts#L82-L91

**Comment by vminhtrung84** [comment]
+1 for this please!
or change pause schedule of the service-blocking to schedule of the service-blocking

**Comment by osokthedevil** [comment]
i second this. i have kid home school and need to keep them on task. would love to see this added and also based on what's being blocked. example gambling site 24/7 block. Roblox from 8 am to 5 pm m-f etc.

**Comment by SavvasMohito** [comment]
+1 for this, I would like to block certain services only 9AM - 5PM, we should be able to do this somehow

**Comment by sofvaskina** [comment]
Thank you for the feature request! A feature like that would indeed be useful for users. I've passed it on to the developers.

**Comment by alexbogun** [comment]
> Thank you for the feature request! A feature like that would indeed be useful for users. I've passed it on to the developers.

@sofvaskina: I’d be willing to work on an implementation and open a pull request, but before starting I’d like to check whether maintainers would be open to accepting this feature. Would keep it backward compatible of course.

**Comment by sofvaskina** [comment]
@alexbogun Thank you! We appreciate your willingness to help.
We’ll be able to handle this on our own, but we sincerely appreciate the offer.

**Comment by TheUkrainian1991** [comment]
+1 for this, it should be one of the features to prioritise. It's been too long since this request was made
